docker_custodian/docker_gc.py

```diff
@@ -334,6 +372,10 @@ def get_args(args=None):
         help="Maxium age for an image. Images older than this age will be "
              "removed. Age can be specified in any pytimeparse supported "
              "format.")
+    parser.add_argument(
+        '--recent-image-age',
+        type=timedelta_type, default=None,
+        help="Images used within specified time interval will not be removed.")
```

Comment:
Can you please add the `Age can be specified in any pytimeparse` helper like the others